portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,15 +27,30 @@
 import com.google.gson.Gson;
 import java.util.*;
 
+/** 
+ * Servlet that stores/returns comments to/from a data store.
+ * @author shradha-khapra
```

Comment:
Author information should only be in file level comments at the top of the file.

Reply:
Got it.